pagopa/io-app
# chore: [IOCOM-1550] New empty archive message

## Short description
This PR changes the empty archive IT message

## List of changes proposed in this pull request
- Empty archive IT translation changed to the desired one

## How to test
Using the io-dev-api-server, set its configuration to archive no messages. Switch to the new messages home, new DS, archive tab and check the message.
